wdn/templates_4.1/less/states/search.less

```diff
@@ -6,15 +6,15 @@
         .nav-scrolling & {
             position: fixed;
             z-index: (@nav-z-index + 1);
-            top: 10px;
+            top: 0;
             bottom: auto;
         }
 
         .nav-scrolling #wdn_menu_toggle:checked ~ #header & {
             left: 100%;
         }
 
-        .wdn_search_toggle:checked ~ #wdn_search_form {
+        .wdn-dropdown-widget-button[aria-pressed='true'] ~ #wdn_search_form {
```

Comment:
Less probably fixes this, but this is not a valid selector. Single quotes cannot be used in attribute selectors.